css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  background-color: rgb(25, 150, 218);
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-full {
+  text-align: center;
+}
```

Comment:
You should make this class something like `text-center`, as of right now the name doesn't match its purpose.